samples/web/content/apprtc/room.py

```diff
@@ -65,6 +65,27 @@ def has_room(host, room_id):
   room = memcache_client.get(key)
   return room is not None
 
+def remove_room(host, room_id, client_id):
+  memcache_client = memcache.Client()
```

Comment:
I thought it's done through some sort of = operator overriding, so that value=None will keep the timestamp.
But honestly I'm not quite sure.
According to http://neopythonic.blogspot.com/2011/08/compare-and-set-in-memcache.html, it works for value+1, which indicates + operator overriding (?). But it could be different for '='.
Anyway, it might be a good idea to invalidate the room, to be safe, e.g. set the room id to None, and let it be evicted when expired.